VerilogCodingStyle.md

```diff
@@ -2385,9 +2407,10 @@ a check for `X` on the 4-state net before resolving to a 2-state variable.
 ***Use logical constructs for logical comparisons, bit-wise for data.***
 
 Logical constructs (`!`, `||`, `&&`, `==`, `!=`) should be used for all
-constructs that are evaluating logic (true or false) values, such as
-if clauses and ternary assignments.  Use bit-wise constructs (`~`, `|`,
-`&`, `^`) for all data constructs, even if scalar.
+constructs that are evaluating control path logic. This includes boolean
```

Comment:
I think the suggestion was that:
1) Logical operators should only be allowed outside if clauses and ternary assignments if they describe control path logic, and
2) A comment should be added saying that this statement is indeed describing control path logic (to clarify the designer's intent).

This was proposed here: https://github.com/lowRISC/style-guides/issues/21#issuecomment-598493498